packages/lambdas/src/document-query.ts

```diff
@@ -49,21 +50,12 @@ function constructErrorResponse(
 
 // Function to construct success response.
 function constructSuccessResponse(
-  payload: DocumentQueryRequestIncoming
-): DocumentQueryResponseOutgoing {
-  const documentReference: DocumentReference = {
-    homeCommunityId: "1.2.3.4.5.6.7.8.9",
-    docUniqueId: "123456789",
-    urn: "urn:oid:1.2.3.4.5.6.7.8.9",
-    repositoryUniqueId: "123456789",
-    contentType: "application/pdf",
-    url: "http://example.com/document.pdf",
-  };
-
+  payload: DocumentQueryReqFromExternalGW
+): DocumentQueryRespToExternalGWSuccessful {
   return {
     id: payload.id,
     timestamp: payload.timestamp,
     responseTimestamp: new Date().toISOString(),
-    documentReference: [documentReference],
+    extrinsicObjectXmls: [""],
```

Comment:
I see this on the Notion page, but I can't find anywhere in our code logic processing this property, nor I could find anything online for it (literally 0 results - also looked [here](https://profiles.ihe.net/ITI/TF/Volume2/ITI-38.html)).

Are we sure we need to change this? Or, if we do, shouldn't this follow an update on the IHE GW?

Current place where I think this is being processed: [link](https://github.com/metriport/metriport/blob/2856886da518317435829519dd40b02e829b094e/packages/ihe-gateway/server/CodeTemplates/XCA%20Library%20-%20Inbound/Get%20XCA%20ITI-39%20RetrieveDocumentSetResponse/Get%20XCA%20ITI-39%20RetrieveDocumentSetResponse.js#L16).

Reply:
All this logic is updated in Jonah's PR 

https://github.dev/metriport/metriport/blob/c027cb45c033c26193d85e167c01ce445525be35/packages/lambdas/src/document-query.ts#L1-L19

Its just being block by the schemas